charts/postgres/templates/migration-job.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: migration-job
+spec:
+  template:
+    metadata:
+      name: migration-job
+    spec:
+      containers:
+      - name: swoop-db
+        image: quay.io/element84/swoop-db
+        command: ["python", "/opt/swoop/db/scripts/run-migration-job.py"]
+        args: ['--action', {{.Values.migration.action}}, '--version', {{.Values.migration.version | quote}}, '--wait_override', {{.Values.migration.wait_override | quote}}]
```

Comment:
We should find a way to pass these via ENV variables to be consistent with other swoop scripts and containers.